.github/workflows/tests-runner.yml

```diff
@@ -0,0 +1,195 @@
+# Executes all the tests on all the platforms
```

Comment:
Does this replace tests.yml+run-tests.yml? How is this being used?

Reply:
Yes, that's the plan to use this instead of the other existing workflows.
Right now, this is to be kicked off manually, since this runsheet isn't running the template tests (per your request). We can enable running this alongside with the existing workflow (to test the templates) but we'll need to remove all other tests from test.yml. Essentially, this is the follow up work.

Reply:
Could you please help me understand why we are repeating the stuff from tests.yml, and run-tests.yml here? Can they not be re-used with the some relevant command line changes?

Reply:
This is supposed to replace tests.yml+run-tests.yml, however we can't do it right now until the Templates tests are also moved to the runsheet.

Reply:
Shouldn't we be able to re-use run-tests.yml as-is, instead of splitting and duplicating it here, and then debugging the new setup again? That way we keep using the existing stuff and replace the test generation stuff with the runsheets.

Reply:
The existing implementation does a lot more, and I have don't want to spend time trying to selectively clean or refactor it to ensure the Template tests continue to run.
I need this change so we can verify it and continue to build on top of it.

Reply:
It creates extra work to review the changes here and in following PRs, which are partially repeating the existing working implementation without being sure that it works. I don't understand the reason for throwing out the working thing.

I'll approve here, and we can discuss when you enable this, I guess.